pkg/cnab/config-adapter/stamp.go

```diff
@@ -64,7 +64,7 @@ func (c *ManifestConverter) GenerateStamp() (Stamp, error) {
 	stamp := Stamp{}
 
 	// Remember the original porter.yaml, base64 encoded to avoid canonical json shenanigans
-	rawManifest, err := manifest.ReadManifestData(c.Context, c.Manifest.ManifestPath)
+	rawManifest, err := manifest.ReadManifestData(c.Context, config.Name)
```

Comment:
Ah yes.  Now that we frequently load Porter's canonical manifest (`build.LOCAL_MANIFEST`/`.cnab/app/porter.yaml`), which sets the corresponding `ManifestPath` field on the manifest in memory, the stamp logic was using this instead of the user's manifest.  I went the approach here of hard-coding to the user's manifest (`config.Name`), since this is the version we'd like to monitor for changes (to spark auto-rebuilds, etc.) but we could also go the route of updating `p.Manifest.ManifestPath` whenever we load the canonical version?  Or, there might be another approach? WDYT?

Reply:
Nope, that's fine. I just wasn't immediately getting the difference between the two.

* ManifestPath = the filepath that we loaded, which could be our rendered .cnab/porter.yaml
* config.Name = always the user's ./porter.yaml